src/main/java/org/toradocu/extractor/ThrowsTag.java

```diff
@@ -96,12 +68,10 @@ public int hashCode() {
    */
   @Override
   public String toString() {
-    String result = super.getKind() + " " + exception + " " + super.getComment();
-    if (super.getCondition() != null
-        && super.getCondition().isPresent()
-        && !super.getCondition().get().isEmpty()) {
-      result += " ==> " + super.getCondition().get();
-    }
-    return result;
+    StringJoiner joiner = new StringJoiner(" ");
+    joiner.add(super.getKind().toString());
+    joiner.add(exception.getName());
```

Comment:
Why does this use `exception` instead of `getException()`?
If you have accessor methods, you might as well use them consistently or document when each one should be used.